bin/coreos-install

```diff
@@ -22,10 +34,7 @@ fi
 : ${VERSION_ID:=current}
 CHANNEL_ID=${GROUP:-stable}
 
-if [[ -e /usr/share/coreos/release ]]; then
-    BOARD=$(gawk --field-separator '=' '/COREOS_RELEASE_BOARD=/ { print $2 }' /usr/share/coreos/release)
-fi
-BOARD=${BOARD:-"amd64-usr"}
```

Comment:
This prevents `BOARD` from being passed in via the environment on non-CL systems, but from the history (54c12a15367c65fc5837e7e1fe699095a482f2ff) I don't think that was an intentional feature.